### ISSUE TITLE
getKey is undefined in Redis Store

### ISSUE THREAD

**Issue by Rohit3523** [express-rate-limit/express-rate-limit#404]
### Description

When i am trying to get a key from ratelimit with getKey function, I am getting `ratelimit.getKey` is not a function

Here is the example code
```
const authRequestRateLimit = rateLimit({
    max: 10,
    windowMs: 60000 * 10,
    keyGenerator: (req) => {
        return `authRequestRateLimit:${req.ip}`
    },
    store: new RedisStore({
        sendCommand: (...args) => client.sendCommand(args)
    })
});

authRequestRateLimit.getKey("authRequestRateLimit:127.0.0.1")
```

When i logged authRequestRateLimit, It is showing getKey: undefined
```
[AsyncFunction (anonymous)] {
  resetKey: [Function: bound resetKey] AsyncFunction,
  getKey: undefined
}
```

### Library version

7.0.2

### Node version

18.13.0

### Module system

CommonJS

**Comment by nfriedly** [comment]
Apologies; `getKey` was a recent addition and it depends on store support. I believe only the memoryStore supports it right now. I'm going to update the readme to make that more clear. 

Would you like to send in a PR to add it to the redis store?

**Comment by Rohit3523** [comment]
I would like to add support for getKey in redis store but i am not sure how the store thing work

**Comment by nfriedly** [comment]
Ok, I'll see if I can do it, but it probably won't be today.

**Comment by Rohit3523** [comment]
Okay will wait for the update 🙂

**Comment by Rohit3523** [comment]
I guess you will have to change lot of things in redis because it only store hitCount in redis, not the resetTime

**Comment by Rohit3523** [comment]
Any update?

**Comment by nfriedly** [comment]
https://github.com/express-rate-limit/rate-limit-redis/pull/195

**Comment by Rohit3523** [comment]
Noticed a minor bug, It is returning NaN instead of 0 if the store doesn't have the key we are passing

**Comment by gamemaker1** [comment]
Sorry about that, I released v4.1.1 with a fix for the issue.

**Comment by Rohit3523** [comment]
Thanks for the quick fix 🙂
